test/library/standard/Sort/errors/errors-comparator.keyargs.good

```diff
@@ -1,2 +1,2 @@
 errors-comparator.chpl:9: In function 'main':
-errors-comparator.chpl:17: error: The comparator keyargs requires a 'key(a)', 'compare(a, b)', or 'keyPart(a, i)' method for element type int(64)
+errors-comparator.chpl:17: error: The comparator keyargs must implement either 'keyComparator', 'keyPartComparator', or 'relativeComparator' for int(64)
```

Comment:
Ditto here and the two remaining .good files - I think this error should talk about keyargs's key method not being correctly findable, not implementing an interface which it does implement.